README.md

```diff
@@ -18,6 +18,123 @@ to facilitate unit-testing of OpenTracing Java instrumentation.
 
 Packages are deployed to Maven Central under the `io.opentracing` group.
 
+## Usage
+
+### Initialization
+
+Initialization is OpenTracing-implementation-specific. Generally speaking, the pattern is to initialize a `Tracer` once for the entire process and to use that `Tracer` for the remainder of the process lifetime. The [GlobalTracer](https://github.com/opentracing/opentracing-java/blob/master/opentracing-util/src/main/java/io/opentracing/util/GlobalTracer.java) provides a helper for singleton access to the `Tracer`.
+
+### `ActiveSpan`s, `Continuation`s, and within-process prapagation
+
+For any execution context or Thread, at most one `Span` may be "active". Of course there may be many other `Spans` involved with the execution context which are (a) started, (b) not finished, and yet (c) not "active": perhaps they are waiting for I/O, blocked on a child Span, or otherwise off the critical path.
+ 
+It's inconvenient to pass an active `Span` from function to function manually, so OpenTracing provides an `ActiveSpanSource` abstraction to provide access to an `ActiveSpan` and to pin and `capture()` it for re-activation in other execution contexts (e.g., in an async callback).
```

Comment:
This is a good question, we need to have a proposal how forks should be handled. Forks are more difficult because OT is using a span model, not an event model. My first reaction is that because we are not using event model, the span should be fundamentally limited to a single execution context at a time, following the span definition as "an operation with duration". Thus in case of a new parallel thread being started by a fork, it only needs to refer to the parent span via parent span context, the Active Span mechanism should not be used.